File: atividade_5/java/src/Calculadora.java
import java.util.Scanner;

public class Calculadora {
    public static void main(String[] args) {
        try (var scan = new Scanner(System.in)) {
            System.out.print("Digite o 1° número: ");
            var n1 = scan.nextDouble();
            System.out.print("Digite o 2° número: ");
            var n2 = scan.nextDouble();
            
            System.out.printf("Soma entre %.1f e %.1f é: %.1f%n", n1, n2, n1 + n2);
            System.out.printf("Subtração entre %.1f e %.1f é: %.1f%n", n1, n2, n1 - n2);
            System.out.printf("Multiplicação entre %.1f e %.1f é: %.1f%n", n1, n2, n1 * n2);
            System.out.printf("Divisão entre %.1f e %.1f é: %.2f%n", n1, n2, n1 / n2);
        } catch (Exception e) {
            System.out.println(e.getMessage());
        }
    }
}

File: atividade_5/C/calculadora.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    double num = 0;
    double num2 = 0;
    printf("Digite digite o 1 número: ");
    scanf("%Lf", &num);
    printf("Digite digite o 2 número: ");
    scanf("%Lf", &num2);
    fflush(stdin);
    printf("A soma entre %.2f e %.2f é %.2f\n", num, num2, num + num2);
    printf("A subtração entre %.2f e %.2f é %.2f\n", num, num2, num - num2);
    printf("A multiplicação entre %.2f e %.2f é %.2f\n", num, num2, num * num2);
    printf("A divisão entre %.2f e %.2f é %.2f\n", num, num2, num / num2);
    return 0;
};